applications/StructuralMechanicsApplication/custom_elements/shell_thin_element_3D4N.cpp

```diff
@@ -1982,17 +1982,15 @@ namespace Kratos
 			r_eta /= 2.0;
 			double l_xi = std::sqrt(inner_prod(r_xi, r_xi));
 			double l_eta = std::sqrt(inner_prod(r_eta, r_eta));
-
+                        
 			for (int i = 0; i < 4; i++)
 			{
 				//eqn 5.2.29
-				Vector vec1 = Vector(MathUtils<double>::CrossProduct
-				(data.r_cartesian[i], s_xi));
+                                const Vector vec1 = MathUtils<double>::CrossProduct(data.r_cartesian[i], s_xi);

```

Comment:
IMPORTANT: not all the instances of CrossProduct have to be changed, only the ones for which you get a deprecation warning

Reply:
I know, I added a const